src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorIntegrationTest.groovy

```diff
@@ -462,6 +480,30 @@ class IqPolicyEvaluatorIntegrationTest
       }
   }
 
+  def 'Pipeline build should fail and stop execution when validate server version fails'() {
+    setup: 'global server URL and globally configured credentials'
+      WorkflowJob project = jenkins.createProject(WorkflowJob)
+      configureJenkins()
+
+    when: 'the nexus policy evaluator is executed'
+      project.definition = new CpsFlowDefinition('node {\n' +
```

Comment:
This might be a little easier to read using Groovy's triple quoted string (http://groovy-lang.org/syntax.html#_triple_single_quoted_string)

Reply:
>This might be a little easier to read using Groovy's triple quoted string (http://groovy-lang.org/syntax.html#_triple_single_quoted_string)

ya, I had considered it but left it as is since I haven't seen it used very much here.  It looks like it's used a bit in this project, so will update to that.

Reply:
> It looks like it's used a bit in this project, so will update to that.

Here's an example from my branch:  https://github.com/sonatype/jenkins-nexus-platform-plugin/blob/fe43c742b2c7b7368b1b742e42aee0ceb50b7f70/src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorIntegrationTest.groovy#L108

Reply:
updated to multi-line string usage in 48311ed855396c55f3465376be249d7cdd7f7947